daemon-tests/tests/happy_path.rs

```diff
@@ -523,6 +525,16 @@ async fn retry_rollover_an_open_cfd() {
         "The maker and taker should have the same commit_txid after the first rollover"
```

Comment:
🔧 I would remove all these asserts on the taker's first rollover success because:

- They're irrelevant for the test. We actually want to immediately "forget" this rollover and pretend it never worked.
- They make the test harder to understand.

Reply:
👍